tt_metal/impl/device/device.hpp

```diff
@@ -237,13 +207,15 @@ class Device {
     // machine inf
     float sfpu_inf() const;
 
-    void generate_device_bank_to_noc_tables();
     const JitBuildEnv& build_env() const { return this->build_env_; }
+    // review: only used in tests
```

Comment:
Perhaps prepend `TESTONLY_` to these names? (or some other way to distinguish)

Reply:
not doing at this point